.github/actions/run-pw-tests/action.yml

```diff
@@ -22,9 +22,15 @@ inputs:
   MAILPITURL:
     description: "mailpit uri"
     required: true
-  URL_TO_RUN: 
+  URL_TO_RUN:
     description: "Url which will be passed to testmo where can be found artifacts of the run"
     required: false
+  PW_WORKERS:
+    description: "Playwright workers"
+    required: true
+  PW_RETRIES:
+    description: "Playwright retries"
+    required: true
```

Comment:
If this is required, other jobs, which are using this action can fail: run-test-cron and run-test-manual.

Reply:
@karola312 fixed